decidim-system/app/views/decidim/system/devise/shared/_links.html.erb

```diff
@@ -3,23 +3,23 @@
 <% end -%>
 
 <%- if devise_mapping.registerable? && controller_name != 'registrations' %>
-  <%= link_to "Sign up", new_registration_path(resource_name) %>
+  <%= link_to t(".sign_up"), new_registration_path(resource_name) %>
 <% end -%>
 
 <%- if devise_mapping.recoverable? && controller_name != 'passwords' && controller_name != 'registrations' %>
-  <%= link_to "Forgot your password?", new_password_path(resource_name) %>
+  <%= link_to t(".forgot_your_password?"), new_password_path(resource_name) %>
 <% end -%>
 
 <%- if devise_mapping.confirmable? && controller_name != 'confirmations' %>
-  <%= link_to "Did not receive confirmation instructions?", new_confirmation_path(resource_name) %>
+  <%= link_to t(".did_not_receive_confirmation_instructions?"), new_confirmation_path(resource_name) %>
 <% end -%>
 
 <%- if devise_mapping.lockable? && resource_class.unlock_strategy_enabled?(:email) && controller_name != 'unlocks' %>
-  <%= link_to "Did not receive unlock instructions?", new_unlock_path(resource_name) %>
+  <%= link_to t(".did_not_receive_unlock_instructions?"), new_unlock_path(resource_name) %>
 <% end -%>
 
 <%- if devise_mapping.omniauthable? %>
```

Comment:
I think I wasn't clear in my comment, but I meant to directly delete these lines (from 21 to 25) as they are not used. Can you do that @greenwoodt ? Thanks

Reply:
I understood Andres. Will have it completed.